test/fixtures/child-tax-credit.json

```diff
@@ -4,7 +4,7 @@
     },
     "title": "Get Child Tax Credit",
```

Comment:
This file looks like a content-api artefact, so we should be able to remove it too.